netlify.toml

```diff
@@ -2,4 +2,9 @@
   publish = "public"
 
 [[plugins]]
-  package = "netlify-plugin-gatsby-cache"
\ No newline at end of file
+  package = "netlify-plugin-gatsby-cache"
+
+[[headers]]
+  for = "/*"
+  [headers.values]
+    Referrer-Policy = "no-referrer-when-downgrade"
```

Comment:
@bertrama Do you think this is an appropriate header?

Reply:
Yeah, this seems okay.  I don't think we're concerned about the referer header leaking to the destination site, but being concerned about a 3rd party eaves dropping might be a privacy concern.